### PR TITLE
Add auth pages.

### PR DESCRIPTION
**Pull request by Xameon**
Login and register pages were added.

### CODE REVIEW

File: src/components/Login/Login.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+.loginBtn:active {
+    background-color: white;
+}
+
+.loginInput {
+    border: none;
+    border-bottom: 1px solid black;
+    margin-top: 0.5rem;
+    font-size: 1.2rem;
+    width: 20rem;
```
Comment (GLFV) [inline src/components/Login/Login.css:48]: use vw or vh instead rem

**Review by GLFV — CHANGES_REQUESTED** [review]

File: src/App.js [hunk 2]
```diff
@@ -7,6 +8,26 @@ import PostsPage from './pages/Posts/PostsPage';
 import './App.css';
 
 function App() {
+  const [user, setUser] = useState({});
```
Comment (yevhenprudnik) [inline src/App.js]: Use null instead of {}.

File: src/App.js [hunk 3]
```diff
@@ -7,6 +8,26 @@ import PostsPage from './pages/Posts/PostsPage';
...
+
+  useEffect(() => {
+    (async () => {
```
Comment (yevhenprudnik) [inline src/App.js:15]: Use usual function instead of ifi.

Comment (yevhenprudnik) [reply src/App.js:15]: Optional.

**Review by yevhenprudnik — CHANGES_REQUESTED** [review]

**Review by yevhenprudnik — APPROVED** [review]

**Pull request merged**